### PR TITLE
Double verify

### PR DESCRIPTION
**Pull request by nmohamed**
Sends verification to both admins and user, then emails user when user is fully verified

### CODE REVIEW

**Comment by shrutiyer** [comment]
:+1:

File: routes/index.js [hunk 1]
```diff
@@ -66,35 +69,71 @@ routes.POSTregister = function(req, res, next) {
...
+        else checkAuth(user, res);
+      });
     }
+    res.redirect('/login');
   });
 }
 
+var checkAuth = function (user, res){
+  if(user.adminAuth !== null && user.userAuth !== null){
+    console.log('\n\nyay\n\n');
```
Comment (davidabrahams) [inline routes/index.js:115]: get rid of this

**Comment by aconsilvio** [comment]
👍

**Comment by davidabrahams** [comment]
EVERYONE WATCH OUT, :bus: IS MERGING

**Pull request merged**